convert2rhel/logger.py

```diff
@@ -82,7 +82,9 @@ class LogLevelTask(object):
 
 class LogLevelFile(object):
```

Comment:
Can't we remove this class altogether and make DEBUG level the default?

Reply:
I wouldn't want the customer to default to DEBUG for CLI runs

Reply:
It looks like the original intent was to output debug to the log file always but output debug to the screen only when `--debug` is passed.

Reply:
> I wouldn't want the customer to default to DEBUG for CLI runs. If so we might as well just change all DEBUG to INFO

@SpyTec, the default log level for stdout has been debug all the time:
https://github.com/oamg/convert2rhel/pull/722/files#diff-f82efabf7bb126100afd0472e3d8a5902da6576c6ddf75e3184cce9eb647c52aL116 

However when logging the debug level, there's a condition that diverts the message to a file only if the --debug is not used: https://github.com/oamg/convert2rhel/pull/722/files#diff-f82efabf7bb126100afd0472e3d8a5902da6576c6ddf75e3184cce9eb647c52aL205

I must say that it's unnecessarily convoluted solution.

> It looks like the original intent was to output debug to the log file always but output debug to the screen only when --debug is passed.

That's true. But I don't think we need a special class to make it happen. I'll try drafting a solution without using the LogLevelFile.

Reply:
https://github.com/oamg/convert2rhel/pull/741